views/show.blade.php

```diff
@@ -84,12 +84,12 @@
 @stop
 
 @section('css')
-{!! Asset::styles('logviewer') !!}
+    {!! HTML::style(asset('styles/logviewer.css') !!}
```

Comment:
Actually, I removed it here because I thought asset() was somehow defined by lightgear/asset. Missed that it is part of laravel.